packages/calypso-e2e/src/lib/components/full-side-editor-nav-sidebar-component.ts

```diff
@@ -3,7 +3,7 @@ import { Page, Locator } from 'playwright';
 const panel = '.edit-site-navigation-panel';
 const selectors = {
 	exitLink: `${ panel } a.edit-site-navigation-panel__back-to-dashboard`,
-	templatePartsLink: `${ panel } [title="Template Parts"] a`,
+	templatePartsLink: `${ panel } ul ul > li:nth-of-type(3) a`,
```

Comment:
**Suggestion (blocking)**: I think we should use a selector here that keys off of the text, or something a little more resilient than hierarchical DOM structure.

```suggestion
	templatePartsLink: `${ panel } a:has-text("Template Parts")`,
```

We've just found over time that selectors that rely too much on DOM structure & ordering tend to be a bit brittle over time. I've found the [Playwright guide on selectors](https://playwright.dev/docs/selectors#best-practices) to be immensely helpful, and highly recommend it! 🙂

Reply:
Your suggestion is much better. I was a bit concerned about the generic-ness of mine. The lesson here to key off of text has been learned and I'll keep in mind for some other upcoming changes.